andela/travel_tool_front
# #162039447 Fix bug on delete first room of guest house

#### What does this PR do?
- Fix bug on delete the first room of a guest house.

#### Description of Task to be completed?
- Add a delete button for first room of a guest house.

#### How should this be manually tested?
- Clone the repo `git clone https://github.com/andela/travel_tool_front.git`
 - Switch to the project directory `cd travel_tool_front`
 - Check out the **bg-delete-first-room-of-guest-house-162039447** branch `git checkout bg-delete-first-room-of-guest-house-162039447`
- Make yourself a travel admin on the user roles table.
- Navigate to the Residence => Manage then click on `Edit Guest House` Button.
- The first room should have a delete button.
- Add one if it doesn't have any rooms.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162039447](https://www.pivotaltracker.com/story/show/162039447)

#### Screenshots (if appropriate)
<img width="1439" alt="screen shot 2018-11-19 at 20 14 46" src="https://user-images.githubusercontent.com/12511966/48723494-d2df3500-ec37-11e8-8565-9368b31d7411.png">


#### Questions:
